api/leaderboard/management/commands/update_db.py

```
@@ -279,3 +295,56 @@ def openlake_contributor_update(self):
+                                if span:
+                                    instance["highest_rating"] = int(span.text)
+                            elif th_text == "Rank":
+                                rank_text = td.text.strip()
+                                match = re.search(r'\d+', rank_text)
+                                if match:
+                                    instance["rank"] = int(match.group())
+
+                instance["username"] = ac_user.username
+                updates.append(instance)
+
+            except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)